### ISSUE TITLE
Errors installing OpenTelemetry-SDK v0.9.0

### ISSUE THREAD

**Issue by ericmustin** [open-telemetry/opentelemetry-ruby#494]
👋 just ensuring we record something discussed in the gitter, i'm seeing some errorr messages when attempting to install the opentelemetry-sdk v0.9.0 gem, seems like a dependancy issue with opentelemetry-common that i haven't quite worked out yet.

Reproduction: https://github.com/ericmustin/docker_reproduction

Consistently seeing failures to install the gem with errors like

```
Downloading opentelemetry-sdk-0.9.0 revealed dependencies not in the API or the
lockfile (opentelemetry-common (~> 0.9.0)).
Either installing with `--full-index` or running `bundle update
opentelemetry-sdk` should fix the problem.
```

**Comment by johnnyshields** [comment]
I saw the same thing. I was able to resolve it by manually removing all opentelemetry gems from my Gemfile.lock then `bundle install`

**Comment by fbogsany** [comment]
Weirdly, the repro didn't work for me (or rather, it *did* work, unexpectedly). I just released v0.10.0. Could you please try with that?

**Comment by ericmustin** [comment]
v0.10.0 resolves the issue for me... im happy to call this closed
